src/inputter.py

```diff
@@ -0,0 +1,36 @@
+# TODO create a script that turns a csv file into the same format as the recipes.json file, this should be used to bulk upload new recipes using the inputter.csv file
+
+import csv
+import json
+
+recipe_name = ""
+rows = []
+
+# read the file into this
+with open("dhal_with_roasted_cauliflower.csv", encoding="utf - 8") as FH:
+    reader = csv.reader(FH)
+    # print(header)
+    recipe_name = next(FH)
+    rows = [row for row in reader]
+
+
+# structure the data here
+split_recipe_name = recipe_name.split(",")
+cleaned_recipe_name = split_recipe_name[0]
+
+ingredients = []
+for row in rows:
+    ingredients.extend(row)
```

Comment:
I don't know why you are flattening this here.

in the row you have a two item list

[weight and unit, ingredient]
so take the 0th element and clean it up

here's a hint, iterate over the string, and filter out any non-alphabetic character. if you do this twice, one to keep only the alphabetic, one to get rid of them, you have the unit and the amount